configs/client/forseti_conf_client.yaml.sample

```diff
@@ -0,0 +1,15 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+server_ip: 192.168.0.1
```

Comment:
Should we be hard-coding this?

Reply:
This is a sample file, only used as an example. During deployment, we will take the forseti_conf_client.yaml.in file, replace the value dynamically and generate forseti_conf_client.yaml with the proper server_ip.